tests/endpoints/test_endpoints.py

```diff
@@ -0,0 +1,107 @@
+from app import db as _db
+from app.serotracker_sqlalchemy.models import DashboardSource
+
+
+def test_healthcheck(client):
+    response = client.get('/healthcheck/')
+    status_code = response.status_code
+    data = response.get_json()
+    assert status_code == 200
+    assert type(data) is str
+    assert data == 'The healthcheck endpoint was hit.'
+
+
+# Test get filter options
+def test_get_filter_options(client):
```

Comment:
for all of these tests, is there a way to store them in the same place as the endpoint so that when the endpoint is modified it is natural to modify these too?

Reply:
We could, I would need to modify our test runner but that can be arranged

Reply:
Think this is worthwhile!